lib/beaker/command_factory.rb

```diff
@@ -16,7 +16,10 @@ def assertions
     end
 
     def execute(command, options={}, &block)
-      result = self.exec(Command.new(command), options)
+      options[:prepend_cmds] ? 
```

Comment:
The other option of course is to keep the processing in command_factory.rb because historically calls to execute have not included passing of options directly to Command.new